examples/nextjs/middleware.ts

```diff
@@ -7,3 +7,7 @@ export const middleware = withMiddlewareAuth({
     redirectTo: '/insufficient-permissions'
   }
 });
+
+export const config = {
```

Comment:
For now, I opted to stick with the stock matcher using the config object. I like keeping this closer to the NextJS API.